backend/routes/api/resources.js

```diff
@@ -61,19 +61,14 @@ router.get(
         distanceFromSearchLoc:
           (geolib.getDistance(
             {
-              latitude: resource.location.coordinates[0],
-              longitude: resource.location.coordinates[1]
+              latitude: resource.location.coordinates[1],
+              longitude: resource.location.coordinates[0]
```

Comment:
┈╱╱▏┈┈╱╱╱╱▏╱╱▏┈┈┈
┈▇╱▏┈┈▇▇▇╱▏▇╱▏┈┈┈
┈▇╱▏▁┈▇╱▇╱▏▇╱▏▁┈┈
┈▇╱╱╱▏▇╱▇╱▏▇╱╱╱▏┈
┈▇▇▇╱┈▇▇▇╱┈▇▇▇╱┈┈

Reply:
🤦‍♀️

Reply:
It really makes me sad that it goes `(long, lat)` since I always get it wrong, but I learned they do this supposedly because longitude = x-coordinate and latitude = y-coordinate, so it makes sense that `(long, lat)` then is `(x,y)`. Still frustrating nonetheless

Reply:
i agree 😭me and alan spent so long wondering why javascript wasn't working when it was just this simple error LMAO i love cs